cmd/configschema/go.mod

```diff
@@ -429,7 +429,7 @@ require (
 	github.com/influxdata/go-syslog/v3 v3.0.1-0.20210608084020-ac565dc76ba6 // indirect
 	github.com/influxdata/influxdb-observability/common v0.5.2 // indirect
 	github.com/influxdata/influxdb-observability/influx2otel v0.5.2 // indirect
-	github.com/influxdata/influxdb-observability/otel2influx v0.5.2 // indirect
+	github.com/influxdata/influxdb-observability/otel2influx v0.5.3-0.20230705233630-3cfabd1d00b9 // indirect
```

Comment:
Not strictly required, but any chance we can use a stable release instead?

Reply:
(This is true because the upstream change breaks the API, so this is an exceptional situation.)